spec/mailers/notify_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+RSpec.describe Notify, type: :mailer do
+  let!(:report) { create(:monthly_report, :with_comments, :with_tags) }
+  let(:user) { create(:user) }
+  let(:from) { "システム管理者 "+"#{Settings.mailer[:from]}" } 
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Surrounding space missing for operator `+`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)